docs/source/managed-federation/federated-schema-checks.mdx

```diff
@@ -2,44 +2,33 @@
 title: Federated schema checks
 ---
 
-> To learn about schema checks for non-federated graphs, see [Schema checks](https://www.apollographql.com/docs/studio/schema-checks/).
+> For an introduction to schema checks, see [Schema checks](https://www.apollographql.com/docs/studio/schema-checks/).
 
-Federation enables teams to work independently on implementing services without needing to coordinate on a monolithic schema. However, this increase in autonomy requires controls to ensure that teams respect [defined dependencies](../federation-spec/) ensure that graph composition succeeds.
+Federation enables teams to work independently on subgraphs without needing to coordinate on a monolithic schema. However, this increase in autonomy requires controls to ensure that teams respect [defined dependencies](../federation-spec/) ensure that supergraph composition succeeds.
```

Comment:
```suggestion
Federation enables teams to work independently on subgraphs without needing to coordinate on a monolithic schema. However, this increase in autonomy requires controls to ensure that teams respect [defined dependencies](../federation-spec/) and ensure that supergraph composition succeeds.
```

I think this sentence at least needs an "and", but I find it a bit confusing overall. I think I might honestly just ... remove it and/or just take a little bit of it and add it to the start of the next paragraph? I like the next paragraph better, it's more succinct and better explains what's actually going on.

Reply:
Yeah, ideally this whole article eventually doesn't exist in favor of folding its guidance into the primary schema checks article. I'll tackle all of these suggestions to make it more readable, but it's a bit of a bear